homeassistant/components/reolink/media_source.py

```diff
@@ -50,7 +50,8 @@ def get_host(hass: HomeAssistant, config_entry_id: str) -> ReolinkHost:
     """Return the Reolink host from the config entry id."""
     config_entry = hass.config_entries.async_get_entry(config_entry_id)
     assert config_entry is not None
-    return config_entry.runtime_data.host
+    host: ReolinkHost = config_entry.runtime_data.host
+    return host
```

Comment:
Instead I think we could cast the config entry to a reolink config entry and then get the host. This way we don't force overwrite a type

Reply:
Thanks for the suggestion, indeed looks nicer.